src/Common/Version.props

```diff
@@ -1,6 +1,6 @@
 <Project>
   <PropertyGroup>
-    <AssemblyVersion>1.13.0</AssemblyVersion>
+    <AssemblyVersion>1.14.0</AssemblyVersion>
```

Comment:
This is managed by the team during the release.

```suggestion
    <AssemblyVersion>1.13.0</AssemblyVersion>
```

Reply:
fixed